libs/auth/certs/openssl.cnf

```diff
@@ -0,0 +1,373 @@
+# =============================================================================
+# This is a copy of the default Debian openssl.cnf (/usr/lib/ssl/openssl.cnf),
+# with one modification (search for "VotingWorks's custom cert fields").
+# =============================================================================
+
+#
+# OpenSSL example configuration file.
+# This is mostly being used for generation of certificate requests.
+#
+
+# Note that you can include other files from the main configuration
+# file using the .include directive.
+#.include filename
+
+# This definition stops the following lines choking if HOME isn't
+# defined.
+HOME			= .
+
+# Extra OBJECT IDENTIFIER info:
+#oid_file		= $ENV::HOME/.oid
+oid_section		= new_oids
+
+# System default
+openssl_conf = default_conf
+
+# To use this configuration file with the "-extfile" option of the
+# "openssl x509" utility, name here the section containing the
+# X.509v3 extensions to use:
+# extensions		=
+# (Alternatively, use a configuration file that has only
+# X.509v3 extensions in its main [= default] section.)
+
+[ new_oids ]
+
+# We can add new OIDs in here for use by 'ca', 'req' and 'ts'.
+# Add a simple OID like this:
+# testoid1=1.2.3.4
+# Or use config file substitution like this:
+# testoid2=${testoid1}.5.6
+
+# Policies used by the TSA examples.
+tsa_policy1 = 1.2.3.4.1
+tsa_policy2 = 1.2.3.4.5.6
+tsa_policy3 = 1.2.3.4.5.7
+
+# VotingWorks's custom cert fields
+component = 1.3.6.1.4.1.59817.1
+jurisdiction = 1.3.6.1.4.1.59817.2
+cardType = 1.3.6.1.4.1.59817.3
+electionHash = 1.3.6.1.4.1.59817.4
+
+####################################################################
+[ ca ]
+default_ca	= CA_default		# The default ca section
+
+####################################################################
+[ CA_default ]
```

Comment:
do we need these defaults? Can we remove them?

Reply:
Yeah great q - we can almost certainly pare this file down. Opened a follow-up issue: https://github.com/votingworks/vxsuite/issues/3111